.changeset/wild-students-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": feature
```

Comment:
I don't know if changeset action understands `feature`s ([reference](https://github.com/changesets/action/blob/main/src/utils.ts#L8-L12)) - we might need to go with `minor`

Reply:
huh, this was generated by the changesets CLI. I can change it to minor

Reply:
Fixed in [6a00e67](https://github.com/primer/react/pull/4386/commits/6a00e67e8cdd3b072d88653a1307582f3896c924)